app/views/users/_account.html.erb

```diff
@@ -87,71 +87,70 @@
+    Deleting your account will:
+    <ul class="delete__account">
+      <li>delete your profile, along with your Twitter and/or GitHub associations.
+        This does not include app permissions, which you will have to remove yourself on Twitter/GitHub.
+      </li>
+      <%# TODO: expand the delete messaging later %>
+      <li>delete any and all content you have, such as articles, comments, your reading list or chat messages.</li>
+      <li>allow your username to become available to anyone.</li>
+    </ul>
+    <div class="field">
+      <label for="delete__account__username__field">
+        <b>To delete your account, type in your username:</b>
+      </label>
+      <input type="text" name="delete__account__username__field" id="delete__account__username__field">
+    </div>
+    <div class="field">
+      <label for="delete__account__verification__field">
+        <b>For verification, type the words:</b><i> delete my account</i>
+      </label>
+      <input type="text" name="delete__account__verification__field" id="delete__account__verification__field">
+    </div>
+    <button class="big-action danger-button small-button" id="delete__account__btn" type="submit" disabled="true">
+      DELETE ACCOUNT
+    </button>
+  <% end %>
+</h4>
+<br>
+<script>
+  var deleteAccountBtn = document.getElementById('delete__account__btn')
+  var deleteAccountUsernameInput = document.getElementById('delete__account__username__field')
+  var deleteAcccountVerificationInput = document.getElementById('delete__account__verification__field')
 
-    function bothInputsVerified() {
-      return deleteAccountUsernameInput.value === '<%= @user.username %>' && deleteAcccountVerificationInput.value === "delete my account"
-    }
+  function bothInputsVerified() {
+    return deleteAccountUsernameInput.value === '<%= @user.username %>' && deleteAcccountVerificationInput.value === "delete my account"
+  }
 
-    deleteAccountUsernameInput.addEventListener('input', function () {
-      if (bothInputsVerified()) {
-        deleteAccountBtn.disabled = false;
-        // enable the button via css or removing disabled class
-      } else {
-        deleteAccountBtn.disabled = true;
-      }
-    })
+  deleteAccountUsernameInput.addEventListener('input', function () {
+    if (bothInputsVerified()) {
+      deleteAccountBtn.disabled = false;
+      // enable the button via css or removing disabled class
+    } else {
+      deleteAccountBtn.disabled = true;
+    }
+  })
 
-    deleteAcccountVerificationInput.addEventListener('input', function () {
-      if (bothInputsVerified()) {
-        deleteAccountBtn.disabled = false;
-      } else {
-        deleteAccountBtn.disabled = true;
-      }
-    })
-  </script>
-<% end %>
+  deleteAcccountVerificationInput.addEventListener('input', function () {
+    if (bothInputsVerified()) {
+      deleteAccountBtn.disabled = false;
+    } else {
+      deleteAccountBtn.disabled = true;
+    }
+  })
```

Comment:
these could be simplified with:

```javascript
const disableDeleteAccountBtn = () => { deleteAccountBtn.disabled = !bothInputsVerified() };
deleteAccountUsernameInput.addEventListener('input', disableDeleteAccountBtn);
deleteAcccountVerificationInput.addEventListener('input', disableDeleteAccountBtn);
```

Reply:
I haven't actually changed the js code here, the change is just indenting, so I think js fixes should be a part of another pr.

Reply:
sure! should we make an issue so we don't forget?